src/Libraries/RevitNodes/Elements/Element.cs

```diff
@@ -710,5 +670,108 @@ protected bool IsAlive
 
+        #region Location extraction & manipulation
+
+        /// <summary>
+        /// Update an existing element's location
+        /// </summary>
+        /// <param name="geometry">New Location Point or Curve</param>
+        public void SetLocation(Geometry geometry)
+        {
+            TransactionManager.Instance.EnsureInTransaction(Application.Document.Current.InternalDocument);
+
+            if (this.InternalElement.Location.GetType() == typeof(Autodesk.Revit.DB.LocationPoint))
+            {
+                if (geometry.GetType() == typeof(Autodesk.DesignScript.Geometry.Point))
+                {
+                    Autodesk.DesignScript.Geometry.Point point = (Autodesk.DesignScript.Geometry.Point)geometry;
+                    Autodesk.Revit.DB.LocationPoint pt = (Autodesk.Revit.DB.LocationPoint)this.InternalElement.Location;
+                    pt.Point = point.ToRevitType(true);
+                }
+                else
+                    throw new Exception("Point element required");
+            }
+            else if (this.InternalElement.Location.GetType() == typeof(Autodesk.Revit.DB.LocationCurve) && geometry is Curve)
+            {
+                if (geometry.GetType() == typeof(Curve))
+                {
+                    Curve dynamoCurve = geometry as Curve;
+                    Autodesk.Revit.DB.LocationCurve curve = (Autodesk.Revit.DB.LocationCurve)this.InternalElement.Location;
+                    curve.Curve = dynamoCurve.ToRevitType(true);
+                }
+                else
+                    throw new Exception("Curve element required");
+            }
+            else
+            {
+                throw new Exception("Element does not have a location that can be updated");
+            }
+
+            TransactionManager.Instance.TransactionTaskDone();
+        }
+
+        /// <summary>
+        /// Get an exsiting element's location
+        /// </summary>
+        /// <returns>Location Geometry</returns>
+        public Geometry GetLocation()
+        {
+            if (this.InternalElement.Location.GetType() == typeof(Autodesk.Revit.DB.LocationPoint))
+            {
+                Autodesk.Revit.DB.LocationPoint pt = (Autodesk.Revit.DB.LocationPoint)this.InternalElement.Location;
+                return pt.Point.ToPoint(true);
+            }
+            else if (this.InternalElement.Location.GetType() == typeof(Autodesk.Revit.DB.LocationCurve))
+            {
+                Autodesk.Revit.DB.LocationCurve curve = (Autodesk.Revit.DB.LocationCurve)this.InternalElement.Location;
+                return curve.Curve.ToProtoType(true);
+            }
+            else
+            {
+                throw new Exception("Element location not extractable");
+            }
+        }
+
+        /// <summary>
+        /// Move Revit Element by Vector
+        /// </summary>
+        /// <param name="vector">Translation Vector</param>
+        public void MoveByVector(Vector vector)
```

Comment:
works in manual tests (Automatic mode) it runs only once and is not being executed again. Since the node has been run. Its not being invalidated and therefore re-executed.